base/libgit2/callbacks.jl

```diff
@@ -213,56 +219,41 @@ function credentials_callback(libgit2credptr::Ptr{Ptr{Void}}, url_ptr::Cstring,
     # get credentials object from payload pointer
     creds = nothing
     creds_are_temp = true
```

Comment:
`securezero!` is triggered by finalizer. I don't think there's really anything else we can do and still retain usability and retries - at least until upstream gives us an argument that tells us whether or not the last call succeeded.